docs/kraken-x3-z3-guide.md

```diff
@@ -166,14 +170,16 @@ they will be removed in a future version and are kept for now for backward compa
 | `backwards-rainbow-pulse` | None | ✓ |
 
 
-## The LCD screen (only Z and 2023 models)
+## The LCD screen (only Z, 2023, 2024 models)
 
 _New in git._<br>
 
 Adds support for NZXT Kraken 2023 Standard, Elite
 
 *On 2023 models (standard and Elite), GIF screen mode is no longer supported for firmware versions 2.X (see [#631][`issue-631`]).*
 
+Adds support for NZXT Kraken 2024 Elite, Elite RGB
```

Comment:
... and this one as well.

Reply:
I'm still scratching my head over why I missed these. I want to apologize for all the back and forth but I am just very nervous.

Reply:
_Please_, don't be 😄 I know it's daunting, but it's okay to miss something - that's why we have back and forth reviews. Don't worry, I'm not going to give up on your PR because it's not perfect the first or the hundredth time (and which one is?) or anything.

For reference, I've been publicly chewed out by people from Google with 30y+ exp. The first ten times you're like "oh GOD how could I miss that..." and after that you're like "oh wow, sorry, I do know better, going to fix, thanks." So just relax, please.